graph/patterns/change-notifications.md

```diff
@@ -0,0 +1,30 @@
+# Pattern name
```

Comment:
Did you mean to add this file?

Reply:
good catch. no change -notifications should go to a different PR.